sklearn/tests/test_metaestimators.py

```diff
@@ -54,10 +58,31 @@ def __init__(self, name, construct, skip_methods=(),
                   lambda est: SelfTrainingClassifier(est),
                   skip_methods=['transform', 'inverse_transform',
                                 'predict_proba']),
+    DelegatorData(
+        'Pipeline_with_NestedMetaEstimator',
+        lambda est: Pipeline([
```

Comment:
For reviewer: a case where we delegate to a fitted estimator within the inner meta-estimator (`final_estimator_`)